superset-frontend/packages/superset-ui-core/test/time-comparison/fetchTimeRange.test.ts

```diff
@@ -116,3 +117,54 @@ it('returns a formatted error message from response', async () => {
     error: 'Network error',
   });
 });
+
+it('fetchTimeRange with shift', async () => {
```

Comment:
Same comment to use `test` instead of `it`.

Reply:
Done!